### DISCUSSION TITLE
Bulk update SEO Post Slug

### DISCUSSION THREAD

**Discussion by chaosdrop** [GetPublii/Publii#1519] [Ideas]
When you duplicate posts it makes horrible post slugs.
If you duplicate a whole bunch of  posts you wind up with a mess of awful post slugs that need to be refreshed after your done renaming them. It would be nice if you could just select them all and have an option under the MORE menu to refresh them all automatically.

**Comment by candidexmedia** [comment]
This is a great feature request!
